Content.Server/Atmos/EntitySystems/GasTileOverlaySystem.cs

```diff
@@ -289,8 +277,8 @@ private void UpdatePlayer(IPlayerSession playerSession, EntityQuery<TransformCom
                 }
             }
 
-            RaiseNetworkEvent(ev, playerSession.ConnectedClient);
-            ReturnToPool(ev.RemovedChunks);
+            if (ev.UpdatedChunks.Count != 0 || ev.RemovedChunks.Count != 0)
+                RaiseNetworkEvent(ev, playerSession.ConnectedClient);
```

Comment:
Not 100% certain this is threadsafe.

Reply:
This should just end up using something like `INetManager.ServerSendMessage` which PVS also uses.

Reply:
Assuming Lidgren is threadsafe looks fine to me as far as I looked.